documentstore_migracao/export/sps_package.py

```diff
@@ -468,10 +468,22 @@ def _set_pub_date(self, attrs_by_version, value):
                 new_node = etree.Element(tag)
                 new_node.text = val
                 pubdate_node.append(new_node)
-        existing_pubdate = self.article_meta.find("pub-date")
 
-        if existing_pubdate is not None:
-            existing_pubdate.addnext(pubdate_node)
+        attr_query = " and ".join(
+            [f"@{k}='{v}'" for k, v in attrs]
+        )  # -> "@pub-type='epub' and @attr='value'"
+        already_existing_nodes = self.article_meta.xpath(f"pub-date[{attr_query}]")
+
+        if len(already_existing_nodes) == 0:
+            self.article_meta.append(pubdate_node)
+            return None
+
+        for pubdate_element in already_existing_nodes:
+            parent = pubdate_element.getparent()
+            index = parent.index(pubdate_element)
+            parent.insert(index, pubdate_node)
+            parent.remove(pubdate_element)
```

Comment:
recentemente vi que existe o `parent.replace` (sugestão caso queira usar)
`
replace(self, old_element, new_element)
Replaces a subelement with the element passed as second argument.
`